Jenkinsfile.github

```diff
@@ -11,14 +14,18 @@ pipeline {
     }
 
     environment {
-        IS_STABLE = getBuildIsStable(releaseBranchIsStable: true)
-        BUILD_METADATA = getRpmRevision(isStable: env.IS_STABLE)
         GIT_REPO_NAME = getRepoName()
-        RELEASE_BRANCH_VERSION = getReleaseBranchVersion()
```

Comment:
I removed this by mistake, this is necessary for a conditional in the publish step.